shell.nix

```diff
@@ -71,7 +71,7 @@ in
 mkShell {
   name = "qmk-firmware";
 
-  buildInputs = [ dfu-programmer dfu-util diffutils git pythonEnv ]
+  buildInputs = [ clang-tools dfu-programmer dfu-util diffutils git pythonEnv ]
```

Comment:
If this could be reverted?

Reply:
`qmk cformat` requires `clang-format` which is part of `clang-tools`. I can do this in a separate PR if you feel appropriate but it's such a small change it seemed unnecessary.

(I needed to do it since I use NixOS and therefore am using this `shell.nix` file).